spring-cloud-kubernetes-istio/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
No need to add this manually. Should be auto generated for configuration properties

Reply:
@spencergibb really? who generates that ? because I needed to add that for core